test/README.md

```diff
@@ -8,16 +8,70 @@
 
 ### End-To-End Tests
 
-## Usage
+## Local Usage
```

Comment:
I like this instructions. But to keep things clean and tidy, we should delete the outdated instructions [here](https://github.com/Moonsong-Labs/storage-hub/blob/2fa2da608da53b4f0e6a0b3cd67a7966dc4339ee/README.md#L50) and put a reference to this instructions in the root folder README.md.